pkg/controller/registry/reconciler/configmap.go

```diff
@@ -166,6 +167,7 @@ type ConfigMapRegistryReconciler struct {
 	Lister   operatorlister.OperatorLister
 	OpClient operatorclient.ClientInterface
 	Image    string
+	MuPod    sync.RWMutex
```

Comment:
I don't think this mutex needs to be exported.

Reply:
It is taken out.